documentation/src/main/asciidoc/manual-index.asciidoc

```diff
@@ -361,3 +361,226 @@ parameters through - see <<lucene-indexing-performance>> for more details.
 The MassIndexer uses a forward only scrollable result to iterate on the primary keys to be loaded,
 but MySQL's JDBC driver will load all values in memory; to avoid this "optimization" set
 `idFetchSize` to `Integer.MIN_VALUE`.
+
+
+[[jsr352-integration]]
+=== Integration with JSR-352
```

Comment:
I'm not sure about the general layout of this documentation. These are the main problems I see:

 * We don't have a summary view of every parameter
 * Some parameters are not documented at all (`optimizeAfterPurge`, etc.)
 * Some concepts are explained across multiple sections, making it difficult to grasp them

Maybe the easier way out is to have two parts in the documentation:

 * a very simple "cheat sheet", i.e. a table with a "parameter name" column, a "builder method" column, and a "description" column (something along the lines of https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#table-worker-configuration). Of course, this table should be introduced properly ("Here is a table of... ", "The 'builder method' column contains...")
 * multiple sections providing in-depth documentation about a particular feature (potentially *multiple* parameters):
   * Defining the indexed entities
   * Retrieving the entity manager
   * Indexing in parallel (max threads + partitions + ...)
   * Dealing with stopped jobs (checkpoints)